grpc/conn.go

```diff
@@ -59,6 +64,22 @@ func (c *ReconfigurableClientConn) NewStream(
 func (c *ReconfigurableClientConn) ReplaceConn(conn rpc.ClientConn) {
 	c.connMu.Lock()
 	c.conn = conn
+	if c.resOnTrackCBs == nil {
```

Comment:
@dgottlieb is it a problem that we are accessing `onTrackCBByTrackName`  without taking the `onTrackCBByTrackNameMu` lock ?